templates/sysconfig/freshclam.erb

```diff
@@ -0,0 +1,5 @@
+#
+# Managed by Puppet
+#
+
```

Comment:
Can you include the standard header like the other template?
https://github.com/broadinstitute/puppet-clamav/blob/525c4dc425a1664a29dcf6fb4033186b9d8b0316/templates/clamav.conf.erb